src/GeoIO.jl

```diff
@@ -131,6 +131,7 @@ include("extra/off.jl")
 include("extra/ply.jl")
 include("extra/stl.jl")
 include("extra/vtk.jl")
+include("extra/projjson.jl")
```

Comment:
Let's move this file to `src/projjson.jl`

Reply:
no problem, but just fyi this is now only called from
```
function projjsonstring(code)
  wktdict = epsg2wktdict(code)
  jsondict = wkt2json(wktdict)
  JSON3.write(jsondict)
end
```
at `src/extras/gis.jl`. I'm not sure if that is the right location for `projjsonstring`.